DQM/SiStripMonitorClient/scripts/LegendToQT.py

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/python
+
+import Image, sys
+
+background = Image.open(sys.argv[1])
+foreground = Image.open(sys.argv[2])
+
+background.paste(foreground, (3000, 40), foreground)
```

Comment:
and finally - i had an outstanding question on an old PR about why this is needed (rather than just modifying the root macro to add the legend)

Reply:
hi- sorry for my late answer - is this not essentially what you need? (modulo a correct color scheme) @chenxvan @fioriNTU 

![numberinactivehits](https://user-images.githubusercontent.com/5042883/30586057-0640525c-9d2f-11e7-8ddb-e368a0f6c3ff.png)

Reply:
yes it is, so how did you obtain this?